app/views/user_mailer/challenge_assignment_notification.html.erb

```diff
@@ -1,13 +1,11 @@
 <% content_for :message do %>
-  <p>
-    You have been assigned the following request in the <%= style_link(@collection.title, collection_url(@collection)) %> challenge at the Archive of Our Own!
-  </p>
+  <p><%= t 'user_mailer.challenge_assignment_notification.html.part1', link: style_link(@collection.title, collection_url(@collection)) %></p>
 
   <p>
-    <%= style_bold("Recipient:") %> <%= @request.nil? ? "None: contact a moderator for help!" : style_link(@request.pseud.byline, user_pseud_url(@request.pseud.user, @request.pseud)) %>
+    <%= style_bold(t('user_mailer.challenge_assignment_notification.recipient')+":") %> <%= @request.nil? ? t('challenge_assignment_notification.recipient_missing') : style_link(@request.pseud.byline, user_pseud_url(@request.pseud.user, @request.pseud)) %>
```

Comment:
I noticed that james has been including the colons in the strings that will be translated -- could you do that as well, please? Or if that's not considered good practice, let me know, and we can ask james to change his pull requests so we're being consistent in our approach.